packages/appstore/src/components/app.tsx

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import { setWebsocket, routes } from '@deriv/shared';
 import { StoreProvider, observer } from '@deriv/stores';
 import CashierStoreProvider from '@deriv/cashier/src/cashier-providers';
+import CFDStoreProvider from '@deriv/cfd/src/cfd-providers';
```

Comment:
There are some code smells in your PR. Ignore "Cognitive Complexity" issues and fix the rest of them please.